.github/workflows/ubuntu_build.yml

```diff
@@ -0,0 +1,66 @@
+name: Calculix 2.17 with preCICE adapter test build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-20.04]
+        include:
+          - os: ubuntu-18.04
+            name: bionic
+          - os: ubuntu-20.04
+            name: focal
+#          - os: ubuntu-21.04
+#            name: hirsute
```

Comment:
You can remove this. In any case, 21.04 will soon reach end-of-life. If you want to test against a non-LTS, try 21.10.

```suggestion
```